TEMPLATE_config.py

```diff
@@ -0,0 +1,6 @@
+# config.py
+# save a copy of this template file as config.py with your BOT TOKEN from discord app settings 
+# and your Giphy api key, both are strings and so need to be in quotes
+
+bot_token = "YOUR_CLIENT_SECRET_GOES_HERE"
```

Comment:
The `client_secret` doesn't go here. The bot token goes here. I would update for clarification since the client_secret is also something that discord uses.